src/query.rs

```diff
@@ -0,0 +1,77 @@
+//! Constants corresponding to standard HTTP names.
+
+/// Common constants for HTTP headers
+pub mod header {
+    /// Common values of HTTP [`Accept`] header.
+    ///
+    /// [`Accept`]: https://datatracker.ietf.org/doc/html/rfc7231#section-5.3.2
+    pub mod accept {
+        /// [JSON]-formatted data.
+        ///
+        /// [JSON]: https://datatracker.ietf.org/doc/html/rfc7159
+        pub const ACCEPT_APPLICATION_JSON: &str = "application/json";
+    }
+
+    /// Common values of HTTP [`Content-Type`] header.
+    ///
+    /// [`Content-Type`]: https://datatracker.ietf.org/doc/html/rfc7231#section-3.1.1.5
+    pub mod content_type {
+        /// [Form-based] data encoded into a URL.
+        ///
+        /// [Form-based]: https://www.ietf.org/rfc/rfc1867
+        pub const X_WWW_FORM_URLENCODED: &str = "application/x-www-form-urlencoded";
+    }
+}
+
+/// Constants corresponding to URL parameter names
+/// and their common values for OAuth2 implementation.
+pub mod param {
+    /// The client identifier issued to the client during the registration process.
```

Comment:
Let's also integrate the value into the docstrings: "`client_id`, the client identifier issued to the client during the registration process".

Reply:
I haven't included them explicitly as rustdoc seems to display constants' values:
![image](https://user-images.githubusercontent.com/7693005/122232033-9cd3c180-cec3-11eb-80a1-e63551dc919a.png)
Yet, I have applied this change in c26d60f.